### ISSUE TITLE
Define and explain the project schedule (ETA)

### ISSUE THREAD

**Issue by Reinmar** [ckeditor/ckeditor5#24]
Keeping in mind that:
- defining 1.0's ETA without explaining the scope of this release will be pointless,
- we should release early and release often, what means that there will be smaller milestones and the solution will be usable in some specific, limited cases earlier than 1.0.0

we should define at some point (the sooner the better) a roadmap leading to the version 1.0.0. We should explain each milestone and what kind of stability and feature completeness we expect to reach at that point.

**Comment by Reinmar** [comment]
We finally came to a point when we can start planning the future with a reasonable precision. We're very close to finish version 0.1.0 which will bring the first demo-able editor. We also defined goals for 2 next sprints.

Check more in https://github.com/ckeditor/ckeditor5/milestones

PS. Every [`ckeditor5-*`](https://github.com/ckeditor?utf8=%E2%9C%93&query=ckeditor5) package (and `ckeditor5` itself) is versioned with its own pace (see [Semantic Versioning](http://semver.org)). E.g. one day we may release 3.0.0 version of `ckeditor5-engine` and 1.5.2 of `ckeditor5-basic-styles`.

This means, that in sprint N we may be working on different versions of different packages. What’s more – we will actually define whether we need to release next major, minor or patch version after a sprint, based on changes that’s been done. So we cannot use one “x.y.z” milestone to mark all tickets in all repositories. We need sprint numbers for milestone names – hence "sprint 2", "sprint 3" and so on.
